runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesOverlay.java

```diff
@@ -64,7 +64,7 @@ public Dimension render(Graphics2D graphics)
 
 			panelComponent.setPreferredSize(new Dimension(
```

Comment:
This is not how this should be done. The panel has padding property (or something like that)

Reply:
I changed nothing but the value of which it was originally created with.

I figured there would be some sort of fontHeight method to refer to, though, it only has one for the width calculation on it's font metrics.

Reply:
Check methods on PanelComponent, there is no need to use absolute sizes at all. Also, this is probably problem at other places too, so fixing it only here do not helps much.